opm/common/data/SimulationDataContainer.cpp

```diff
@@ -77,11 +87,57 @@ namespace Opm {
             return iter->second;
     }
 
+    const std::vector<double>& SimulationDataContainer::getFaceData( const std::string& name ) const {
+        auto iter = m_face_data.find( name );
+        if (iter == m_face_data.end())
+            throw std::invalid_argument("The Face data with name: " + name + " does not exist");
+        else
+            return iter->second;
+    }
+
     void SimulationDataContainer::registerFaceData( const std::string& name , size_t components , double initialValue) {
         if (!hasFaceData( name ))
             m_face_data.insert( std::pair<std::string , std::vector<double>>( name , std::vector<double>(components * m_num_faces , initialValue )));
     }
 
+    bool SimulationDataContainer::equal( const SimulationDataContainer& other ) const {
+        if ((m_num_cells != other.m_num_cells) ||
```

Comment:
Please add braces to all `if` blocks, even if they only contain a single statement.